owtf/models/command.py

```diff
@@ -17,7 +17,7 @@ class Command(Model):
     end_time = Column(DateTime)
     success = Column(Boolean, default=False)
     target_id = Column(Integer, ForeignKey("targets.id"))
-    plugin_key = Column(String, ForeignKey("plugins.key"))
+    plugin_key = Column(String)
```

Comment:
@viyatb is there any other way we could solve the `Foreign key associated with column 'command_register.plugin_key' could not find table 'plugins'` error. This seems to be a good temporary fix but might cause problem elsewhere in the code. I tried a bunch of things but nothing worked out.

Reply:
Rest of the changes seems fine.

Reply:
This is solved at #1040 Thanks!